installation.rst

```diff
@@ -3,3 +3,166 @@
 Install
 =======
 
+Conan can be installed in many Operating Systems. It has been extensively used and tested in Windows, Linux (different distros), OSX, and is
+also actively used in FreeBSD and Solaris SunOS. There are also several additional operating systems on which it has been reported to work.
+
+There are three ways to install Conan:
+
+1. The preferred and **strongly recommended way to install Conan** is from PyPI, the Python Package Index, using the ``pip`` command.
+2. There are other available installers for different systems, which might come with a bundled python interpreter, so that you don't have to
+   install python first. Note that some of **these installers might have some limitations**, especially those created with pyinstaller
+   (such as Windows exe & Linux deb).
+3. Running Conan from sources.
+
+Install with pip (recommended)
+------------------------------
+
+To install Conan using ``pip``, you need a Python >= 3.5 distribution installed on your machine. Modern Python distros come with pip
+pre-installed. However, if necessary you can install pip by following the instructions in `pip docs`_.
+
+.. warning::
+    Python 2.x and Python <= 3.4 support has been dropped. Conan will not work with those python versions.
+
+Install Conan:
+
+.. code-block:: bash
+
+    $ pip install conan
```

Comment:
Probably we want to investigate ``pipx``. I have tried it and works pretty well, it might highly simplify the isolated installation. Not for now, just a note.